client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.jsx

```diff
@@ -85,10 +98,17 @@ const DataFilesProjectMembers = ({
   const columns = [
     memberColumn,
     {
-      Header: 'Access',
-      accessor: 'access',
+      Header: 'Role',
+      accessor: 'user.username',
+      id: 'role',
       className: 'project-members__cell',
-      Cell: (el) => <span className={styles.access}>{el.value}</span>,
+      Cell: projectId
+        ? (el) => (
+            <SystemRoleSelector projectId={projectId} username={el.value} />
+          )
+        : (el) => (
+            <span>{mapAccessToRoles(el.row.original.access).systemRole}</span>
+          ),
     },
     {
       Header: loading ? (
```

Comment:
The "Confirm Transfer Ownership" text appears cut off
![Screen Shot 2022-04-27 at 5 34 51 PM](https://user-images.githubusercontent.com/20326896/165642770-1719e590-98d3-41a6-a585-a9021a988228.png)